book/src/integrations/rust.md

```diff
@@ -0,0 +1,37 @@
+# Rust (prql-compiler)
+
+## Installation
+
+```
+cargo new myproject
+cd myproject
+cargo install prql-compiler
```

Comment:
This will install the binary; I think we want this for adding to a crate
```suggestion
cargo add prql-compiler
```

Reply:
Learning every day!